draft-ehlen-openpgp-nist-bp-comp.md

```diff
@@ -563,7 +563,7 @@ TBD (ML-KEM-1024+ECDH-brainpoolP384r1)    | ML-KEM-1024  | ecdhKem   | brainpool
 
 The ML-KEM + ECC composite public-key encryption schemes are built according to the following principal design:
 
- - The ML-KEM encapsulation algorithm is invoked to create a ML-KEM ciphertext together with a ML-KEM symmetric key share.
+ - The ML-KEM encapsulation algorithm is invoked to create an ML-KEM ciphertext together with an ML-KEM symmetric key share.
```

Comment:
```suggestion
 - The ML-KEM encapsulation algorithm is invoked to create an ML-KEM ciphertext together with an ML-KEM shared secret key.
```

The usual KEM terminolofy is "shared secret" and in FIPS 203 ipd they write "share secret key".

Reply:
This might have to be applied to other places as well. I don't object to leaving it as is, but I think "shared secret (key)" would be better.

Reply:
In order not to hold up our submission version, I made an issue for your proposal: https://github.com/openpgp-pqc/draft-ehlen-openpgp-nist-bp-comp/issues/11